core/src/main/java/org/stellar/anchor/util/StringHelper.java

```diff
@@ -24,4 +24,13 @@ public static String camelToSnake(String camel) {
         .replaceAll("-", "_")
         .toLowerCase();
   }
+
+  public static String toUpperSnake(String camel) {
+    return camel
+        .replaceAll("([A-Z]+)([A-Z][a-z])", "$1_$2")
+        .replaceAll("([a-z])([A-Z])", "$1_$2")
+        .replaceAll("-", "_")
+        .replaceAll("\\.", "_")
+        .toUpperCase();
```

Comment:
Not sure how we would call something like `sep1.toml`?  `camel path`?

Can we reuse the `camelToSnake()`?

```suggestion
  public static String camelPathtoUpperSnake(String camel) {
    return camelToSnake(camel)
        .replaceAll("\\.", "_")
        .toUpperCase();
```